README.md

```diff
@@ -172,3 +172,14 @@ Functional tests are written with PHPUnit. Note that Bundles and Components are
 ## Remove demo configuration
 
 If you start a project from the sandbox, remove ```.sensiolabs.yml``` as its not a good example for production use.
+
+## Apply vagrant box
+
+There's a new vagrant box that is simple to boot:
```

Comment:
I would suggest to provide some docs around the sandbox and its install in our common docs.